docs/attributes.md

```diff
@@ -66,3 +66,69 @@ _Example_: Extract `src` and `alt` from each image element in the block's markup
 //   ]
 // }
 ```
+
+## Meta
+
+Attributes may be obtained from a post's meta rather than from the block's representation in saved post content. For this, an attribute is required to specify its corresponding meta key under the `meta` key:
+
+```js
+attributes: {
+	author: {
+		type: 'string',
+		meta: 'author'
+	},
+},
+```
+
+From here, meta attributes can be read and written by a block using the same interface as any attribute:
+
+{% codetabs %}
+{% ES5 %}
+```js
+edit: function( props ) {
+	function onChange( event ) {
+		props.setAttributes( { author: event.target.value } );
+	}
+
+	return el( 'input', {
+		value: props.attributes.author,
+		onChange: onChange,
+	} );
+},
+```
+{% ESNext %}
+```js
+edit( { attributes, setAttributes } ) {
+	function onChange( event ) {
+		setAttributes( { author: event.target.value } );
+	}
+
+	return <input value={ attributes.author } onChange={ onChange } />;
+},
+```
+{% end %}
+
+### Considerations
+
+By default, a meta field will be excluded from a post object's meta. This can be circumvented by explicitly making the field visible:
+
+```php
+function gutenberg_my_block_init() {
+	register_meta( 'post', 'author', array(
+		'show_in_rest' => true,
+	) );
+}
+add_action( 'init', 'gutenberg_my_block_init' );
+```
+
+Furthermore, be aware that WordPress defaults to not treating a meta datum as being unique, instead returning an array of values. If that behavior is not desired, the same `register_meta` call can be complemented with the `single` parameter as follows:
+
+```php
+function gutenberg_my_block_init() {
+	register_meta( 'post', 'author', array(
+		'show_in_rest' => true,
+		'single' => true,
+	) );
+}
+add_action( 'init', 'gutenberg_my_block_init' );
```

Comment:
It may also be worth mentioning that `register_meta` treats the type as `string` by default, which may be surprising behaviour.